kwalify/component/test_data_validity.py

```diff
@@ -0,0 +1,26 @@
+from glob import iglob
+from pykwalify.core import Core
+import yaml
+
+def get_schema(version):
+    path = 'schemas/kwalify/component/v{}.yaml'.format(version)
+    contents = open(path)
+    return yaml.load(contents)
+
+def create_validator(source_data):
+    version = source_data.get('schema_version', '1.0.0')
```

Comment:
if we provide a file without `schema_version`  or it can't be found, 1.0.0 will be found. but there's no `v1.0.0` for component and when it goes to call `get_schema` it will try to open a file that doesn't exist.